From 939cd9e774a42000495921b70e5322c5cec7d474 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stjepan=20Gro=C5=A1?= <stjepan.gros@gmail.com>
Date: Wed, 30 Apr 2014 11:52:35 +0200
Subject: [PATCH 13/17] Temporarily raised hop limit to 10 so that
 DHCPv4-over-DHCPv6 can be tested. Ideally, this should be configuration
 parameter or it should be determined based on the protocol used!

---
 networking/udhcp/d6_packet.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/networking/udhcp/d6_packet.c b/networking/udhcp/d6_packet.c
index 79b2946..4a0fe16 100644
--- a/networking/udhcp/d6_packet.c
+++ b/networking/udhcp/d6_packet.c
@@ -105,7 +105,7 @@ int FAST_FUNC d6_send_raw_packet(
 				offsetof(struct ip6_udp_d6_packet, data) - 4 + d6_pkt_size
 	);
 	/* fix 'hop limit' and 'next header' after UDP checksumming */
-	packet.ip6.ip6_hlim = 1; /* observed Windows machines to use hlim=1 */
+	packet.ip6.ip6_hlim = 10; /* observed Windows machines to use hlim=1 */
 	packet.ip6.ip6_nxt = IPPROTO_UDP;
 
 	d6_dump_packet(d6_pkt);
-- 
2.0.1

